### ISSUE TITLE
Folder zeto/zkp/golang does not exist

### ISSUE THREAD

**Issue by Alexander-Herranz** [hyperledger-labs/zeto#117]
In this [README](https://github.com/hyperledger-labs/zeto/tree/main/zkp/circuits) file, in the section **Run Circuit Tests**, the following bullet is mentioned:

- golang: go to the [golang](https://github.com/hyperledger-labs/zeto/blob/main/zkp/golang) folder

But the folder does not exist.

**Comment by jimthematrix** [comment]
indeed. thanks for catching that. interested in submitting a PR for the update?

**Comment by HMSagar2701** [comment]
Hi @jimthematrix , Can i work on this
